### DISCUSSION TITLE
Selective channel loading by group / channel Index

### DISCUSSION THREAD

**Discussion by onlynow2** [danielhrisca/asammdf#760] [Q&A] [answered]
Hello, 

I've been playing around with selective channel loading of MDF - files  for a while (for converting only the neccesarry channels to a pandas dataframe afterwards).
Regarding the documentation your able to give a list of strings containing the channels to load, e.g.:
`mdf = MDF(name='measurement.mf4', channels=channellist)`

Sometimes channels from different ECUs appear under the same name, but different source within MDF - files (Channel group and channel index differ as well of course).

Is it possible to handle this situation and specify which channel should be loaded?
I think of either aditionally providing the source of the channel or giving a list of group / channel index to load.

Help is much appreciated.
Kind regards

**Comment by danielhrisca** [comment, marked as answer]
Have a look at the documentation for the [to_dataframe](https://asammdf.readthedocs.io/en/master/api.html#asammdf.mdf.MDF.to_dataframe) method

**Reply by onlynow2** [reply]
Thanks for the hint, i'll check this out.
